miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/page/element/DataTable.java

```diff
@@ -77,6 +79,14 @@ public String getTextAtCell(String columnHeading, int rowNum) {
     return cell.getText();
   }
 
+  public void checkBoxForRow(int rowNum) {
+    List<WebElement> checkbox = getCell(Columns.SORT, rowNum).findElements(By.tagName("input"));
+    if (checkbox.isEmpty()) {
+      throw new IllegalArgumentException("Row " + rowNum + " does not have a checkbox to click.");
+    }
+    checkbox.get(0).click();
```

Comment:
Use `findElementIfExists` and null check instead. Includes check for unexpected multiple results

Reply:
In this case, the "find" needs to be only on the results of `getCell`. `findElementIfExists` will look over an entire page, and tables with more than one row will cause it to throw an error.